hazelcast/src/main/java/com/hazelcast/ringbuffer/impl/client/PortableReadResultSet.java

```diff
@@ -136,10 +132,5 @@ public void readPortable(PortableReader reader) throws IOException {
             Data item = rawDataInput.readData();
             items.add(item);
         }
-        seqs = rawDataInput.readLongArray();
-        // RU_COMPAT_3_9
-        if (rawDataInput.getVersion().isGreaterOrEqual(Versions.V3_10)) {
-            nextSeq = rawDataInput.readLong();
-        }
```

Comment:
Conditional serialization won't work here: `seqs` was already being conditionally serialized based on version since 3.9, however the input's version was never set as versioned serialization does not work in portables -> the field wasn't serialized. Same goes for the `nextSeq` field.